mxabierto/adela
# Fix update library

### Changelog

* TODO:  update to Gemfile.lock to avoid vulnerabilites in Adela Project.

### How to test

* TODO: Follow the next steps.

1) Download this PR into your local environment.
2) Run in your console the next two files:

a) rake bundle_audit:run  (To see the actual vulnerabilities)
b) rake bundle_audit:update (update de gemfile.lock)
c) rake bundle_audit:run:  (To verify most of the vulnerabilities has been removed from you gemfil.lock)
d) Start up you systema Adela and navigate using it in order to see everything has the same behavior with the new version of gems.
